src/pages/index.js

```diff
@@ -21,8 +22,8 @@ class Index extends React.Component {
     return (
       <div>
         {index_lang.title[this.state.lang]}
-        <Button action={() => console.log("Button Press!")}>
-          Press me!
+        <Button action={() => console.log("Contact Us!")}>
+          Contact Us
```

Comment:
This is apart of the nav bar, so I will be getting rid of this anyways. But no need to change.

Reply:
Yes, currently the index.html is only for testing out the components that I've made so far, this was just a check to make sure that the button looks right